readthedocs/proxito/views/serve.py

```diff
@@ -673,3 +675,45 @@ def changefreqs_generator():
 
 class ServeSitemapXML(SettingsOverrideObject):
     _default_class = ServeSitemapXMLBase
+
+
+class ServeStaticFiles(CachedView, CacheTagsMixin, View):
+
+    """
+    Serve static files from the same domain the docs are being served from.
+
+    This is basically a proxy for ``STATIC_URL``.
+    """
+
+    project_cache_tag = "staticfile"
+    cache_request = True
+
+    @method_decorator(map_project_slug)
+    def get(self, request, filename, project):
+        try:
+            # This is needed for the _get_project
+            # method for the CacheTagsMixin class.
+            self.project = project
+            return FileResponse(staticfiles_storage.open(filename))
```

Comment:
I do worry a little about this, performance-wise. I think this file will be heavily cached at the CDN though, so I think it's OK for now.

Reply:
I don't follow why we are returning a `FileResponse` here instead of an `HttpResponse` with the `Send-File` header on it instead of as we do for the other files?

Reply:
No strong reason. We will just need to setup a new internal redirect on nginx

Reply:
In that case, I'd vote for using the same technique we are using with all the other file types. For consistency and also because we already know that it works fine and it can serve big files without issues. Using `FileResponse` here breaks the consistency and may produce performance issues that will be harder to debug.

Reply:
@ericholscher thoughts?

Reply:
I'm 👍  on keeping it consistent using Sendfile, so we know how it works.